eth/blockwatch/block_watcher_test.go

```diff
@@ -214,6 +214,34 @@ func TestGetMissedEventsToBackfillSomeMissed(t *testing.T) {
 	assert.Equal(t, big.NewInt(30), headers[0].Number)
 }
 
+func TestGetMissedEventsToBackfill_BackfillStartBlock(t *testing.T) {
+	// Fixture will return block 30 as the tip of the chain
+	fakeClient, err := newFakeClient("testdata/fake_client_fast_sync_fixture.json")
+	require.NoError(t, err)
+
+	store := &stubMiniHeaderStore{}
+
+	config.Store = store
+	config.Client = fakeClient
+	config.BackfillStartBlock = big.NewInt(0)
```

Comment:
Can we also test with a non-zero backfill start block?

Reply:
fixed in be50d19d